litgpt/finetune/lora.py

```diff
@@ -262,10 +263,15 @@ def fit(
                     iter_num * train.micro_batch_size * model.config.block_size * fabric.world_size
                 ),
                 "learning_rate": scheduler.get_last_lr()[0],
+                "val_loss": val_loss,
             }
+            val_loss_str = 'None' if metrics['val_loss'] is None else f"{metrics['val_loss']:.3f}"
             fabric.print(
-                f"iter {metrics['iter']} | step {metrics['step']}: loss {metrics['loss']:.4f}, iter time:"
-                f" {metrics['iter_time'] * 1000:.2f} ms{' (optimizer.step)' if not is_accumulating else ''}"
+                f"Ep {metrics['epoch']+1} | iter {metrics['iter']} | step {metrics['step']}:"
+                f" loss train (curr): {metrics['loss']:.3f},"
+                f" val (prev val iter): {val_loss_str} |"
```

Comment:
My preference is to keep the information compact.
1) I don't think anyone is questioning whether the training loss is from any other iteration than the current one, especially when it is numbered by the iteration in the same line. So I suggest to drop (curr).
2) Same argument for val, why wouldn't it be the value from the previous validation? And we don't validate every training iteration.